src/widgets/search-box/search-box.js

```diff
@@ -59,18 +65,18 @@ const renderer = ({
     const form = input.parentElement;
 
     if (searchAsYouType) {
-      addListener(input, 'input', getInputValueAndCall(refine));
-      addListener(form, 'submit', e => {
-        e.preventDefault();
+      input.addEventListener('input', refine);
+      form.addEventListener('submit', event => {
+        event.preventDefault();
         input.blur();
       });
     } else {
-      addListener(input, 'input', e => {
-        refine(getValue(e), false);
+      input.addEventListener('input', event => {
+        refine(event, false);
```

Comment:
It doesn't work anymore since we provide the event to `refine` rather than the value. It should probably be catch with a test.